edge/pkg/eventbus/dao/subtopics.go

```diff
@@ -0,0 +1,43 @@
+package dao
+
+import (
+	"k8s.io/klog/v2"
+
+	"github.com/kubeedge/kubeedge/edge/pkg/common/dbm"
+)
+
+const (
+	SubTopicsName = "sub_topics"
+)
+
+type SubTopics struct {
+	Topic string `orm:"column(topic); type(text); pk"`
+}
+
+// InsertTopics insert sub_topics
+func InsertTopics(event *SubTopics) error {
+	_, err := dbm.DBAccess.Raw("INSERT OR REPLACE INTO sub_topics (topic) VALUES (?)", event.Topic).Exec()
```

Comment:
The Replace here may can avoid to insert many times 👍 
And will it create a new table in the DB?

Reply:
The topic is the primary key, so it's unique, I changed the input into string here. And .. yes it will create a new table in the DB and the name of the table is sub_topics.

Reply:
okey, the code here will create the table automatically?

Reply:
the table will be createrd automatically due to the follow 
![image](https://user-images.githubusercontent.com/71764707/103391254-6c85aa00-4b53-11eb-8ce9-1666eda685ea.png)